Solutions/moritz_w/Lesson_5/pllsettings.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+import re
+
+
+class PLLSetting:
+    def __init__(self):
+        self.index = None
+        self.frequency = None
+        self.settings = None
+
+
+def main():
+    unittest.TextTestRunner(verbosity=2).run(unittest.TestLoader().discover("."))
```

Comment:
You dont have to copy these tests files. they will run from course location. In this case: "../../../Course/Lesson_5".
unittest.TextTestRunner(verbosity=2).run(unittest.TestLoader().discover("../../../Course/Lesson_5"))

You can add additional tests if you want to in your folder. Then you can run them by adding second line with:
unittest.TextTestRunner(verbosity=2).run(unittest.TestLoader().discover("."))

This is more general comment to your commits.